thirdparty/fetch-thirdparty-deps.ps1

```diff
@@ -130,4 +130,20 @@ if (!(Test-Path "Eluant.dll"))
 	(New-Object System.Net.WebClient).DownloadFile("https://github.com/OpenRA/Eluant/releases/download/20140425/Eluant.dll", $target)
 }
 
+if ((Test-Path "GeoLite2-Country.mmdb"))
+{
+	$lastWrite = (get-item "GeoLite2-Country.mmdb").LastWriteTime
+	$timespan = new-timespan -days 30
+	if (((get-date) - $lastWrite) -lt $timespan)
+	{
+	    Exit
```

Comment:
You probably want to invert the check and then move the download inside this block.  Exiting here will prevent the `cd ..` at the end of the file from executing.